dts/bindings/mtd/jedec,spi-nor.yaml

```diff
@@ -24,3 +24,25 @@ properties:
     type: phandle-array
     required: false
     description: RESETn pin
+  page-size:
+    type: int
+    required: false
+    default: 256
```

Comment:
remove default here, and add this to the various dts files.

Reply:
Why?  The vast majority of devices use 256-byte pages; I think there's exactly model that doesn't.

Reply:
Please see my comment further up about the defaults. One device with different page size is cypress s25fl512s which uses 512 bytes